test/api/test_gpu.cpp

```diff
@@ -167,7 +167,7 @@ TEST_CASE(dynamic_batch_load_and_run_offload)
 
     CHECK(shapes_before.size() == outputs.size());
     CHECK(bool{outputs.front().get_shape() ==
-               migraphx::shape(migraphx_shape_float_type, {2, 1, 3, 3})});
+               migraphx::shape(migraphx_shape_float_type, {2, 2, 3, 3})});
```

Comment:
Why wasn't this failing on the CI ?

Reply:
It looks like using `CHECK` rather than `EXCEPT` doesn't trigger a test failure. I should probably make an issue about that?

Reply:
> I should probably make an issue about that? 

Yes. and also change this test to use EXPECT for now.